src/app/screens/Accounts/index.test.tsx

```diff
@@ -0,0 +1,17 @@
+import { render, screen } from "@testing-library/react";
+import { MemoryRouter } from "react-router-dom";
+import Accounts from "./index";
+import { AccountsProvider } from "../../context/AccountsContext";
+
+describe("Accounts", () => {
+  test("render", async () => {
+    render(
+      <AccountsProvider>
```

Comment:
I think for now this is ok but we should imrpove this:
- Provide a render functions with AccountsProvider and MemoryRouter which can be imported by other tests
- AccountsProvider should provide mocked accounts for further tests

@dylancom hapyy for input/feedback here. I could work on this because I feel like atm I'm blocking releases if I work on high-prio issues.